ironfish/src/fileStores/index.ts

```diff
@@ -4,3 +4,4 @@
 export * from './config'
 export * from './fileStore'
 export * from './internal'
+export * from './hostsStore'
```

Comment:
the base storage is called FileStore because it's an abstract store, but none of the other stores like `config` or `internal` have this `Store` suffix. What do you think about making them consistent?

Reply:
Sure, will do.